docs/baremetalhost-states.md

```diff
@@ -45,6 +45,10 @@ profile.
 
 A host in the Ready state is available to be provisioned.
 
+## Preparing
+
+While a host is being setted capabilities, it will be in Preparing state.
```

Comment:
"while setting capabilities for a host, the host is in Preparing state."

Reply:
Thanks, done.